ApplicationLibCode/SocketInterface/RiaSocketServer.cpp

```diff
@@ -56,15 +56,24 @@ RiaSocketServer::RiaSocketServer( QObject* parent )
     m_nextPendingConnectionTimer->setInterval( 100 );
     m_nextPendingConnectionTimer->setSingleShot( true );
 
-    if ( !m_tcpServer->listen( QHostAddress::LocalHost, 40001 ) )
+    int portNumber = riOctavePlugin::defaultPortNumber;
+    if ( !RiaPreferences::current()->octavePortNumber().isEmpty() )
```

Comment:
Maybe validate some portNumber value more here? No it can be an int outside the valid port range?

Reply:
I do not think this is required, this is a debug/advanced feature for special needs. Default port works for RHEL8